docs/standards/smart-contracts/lsp0-erc725-account.md

```diff
@@ -79,12 +186,24 @@ Return the address of the pending owner that was initiated by [`transferOwnershi
 
 ### transferOwnership
 
+:::info
+
+Check the [**transferOwnership(..)**](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-0-ERC725Account.md#transferownership) function specification in [**LSP0-ERC725Account Standard**](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-0-ERC725Account.md) in the **LIP repository**.
+
+Check the [**transferOwnership(..)**](https://github.com/lukso-network/lsp-smart-contracts/blob/v0.8.0/contracts/LSP0ERC725Account/LSP0ERC725AccountCore.sol#L290) function implementation in **LSP0ERC725Account** Contract.
+
+:::
+
 ```solidity
 function transferOwnership(address newOwner) public
 ```
 
 Initiate an ownership transfer by setting the `newOwner` as `pendingOwner`.
 
+Calls the `universalReceiver(..)` function [**on the pending owner**](https://github.com/lukso-network/lsp-smart-contracts/blob/v0.8.0/contracts/LSP0ERC725Account/LSP0ERC725AccountCore.sol#L328), if it supports LSP1 InterfaceId with the following typeId:
```

Comment:
I suggest that we say `newOwner` for this line, as for new developer it might be easier to understand.

```suggestion
This will also call the `universalReceiver(..)` function [**of the `newOwner`**](https://github.com/lukso-network/lsp-smart-contracts/blob/v0.8.0/contracts/LSP0ERC725Account/LSP0ERC725AccountCore.sol#L328), if it supports LSP1 InterfaceId with the following typeId:
```

Reply:
Actually, i think naming the param `newOwner` is confusing, because it's not the newOwner until it acceptOwnership. So I suggest keeping the pendingOwner name, and renaming the parameter to something like potentielOwner or pendingOwner